packages/office-ui-fabric-react/src/components/DetailsList/index.ts

```diff
@@ -1,4 +1,3 @@
-export * from '../../utilities/selection/index';
```

Comment:
We'll want to check ODSP to see how they reference `Selection` to ensure modifying this export doesn't break AMD imports for them.